.gitignore

```diff
@@ -8,8 +8,11 @@
 /promtool
 benchmark.txt
 /data
+/data-agent
 /cmd/prometheus/data
 /cmd/prometheus/debug
+/cmd/prometheus-agent/debug
+/cmd/prometheus-agent/data-agent
```

Comment:
Right now there isn't a /cmd/prometheus-agent directory, should this just be excluding /cmd/prometheus/data-agent?

Reply:
Thanks, forgot to change this when I combined the two main.gos.